lib/modules/dosomething/dosomething_kudos/includes/KudosTransformer.php

```diff
@@ -23,6 +23,7 @@ public function index($parameters) {
       $kudos = services_resource_build_index_list($kudos, 'kudos', 'id');
     }
     catch (Exception $error) {
+      http_response_code('404');
```

Comment:
@DFurnes I removed this for index but should keep this error code here for the `show`, right?